### DISCUSSION TITLE
Other Palette?

### DISCUSSION THREAD

**Discussion by ghost** [SylEleuth/gruvbox-plus-icon-pack#117] [General]
Anyone or the author may provide us another palette apart gruvbox?

I would really like someone to fork this repo and continue it as original [Suru](https://github.com/snwh/suru-icon-theme), or [Suru++](https://github.com/gusbemacbe/suru-plus).

**Comment by SylEleuth** [comment]
I'm not going to do anything with that beyond Gruvbox colour scheme but changing colours of monochromatic icons would be easy. The most challenged would be scalable icons. You could automate it to have something on the start (that's what I did) but polishing it would take months. And then maintaining it maybe a lifetime.

**Comment by ghost** [comment]
I understand, I thought someone has it easy with some tools or scripts that make it easy transformation.
